modules/sdk-coin-trx/src/trx.ts

```diff
@@ -792,27 +792,70 @@ export class Trx extends BaseCoin {
       const accountInfo = await this.getAccountBalancesFromNode(receiveAddress);
 
       if (accountInfo.data[0] && accountInfo.data[0].balance > SAFE_TRON_TRANSACTION_FEE) {
-        const addressBalance = accountInfo.data[0].balance;
-        const addressInfo = {
-          address: receiveAddress,
-          chain: 0,
-          index: i,
-        };
-        const recoveryAmount = addressBalance - SAFE_TRON_TRANSACTION_FEE;
-        const buildTx = await this.getBuildTransaction(baseAddrHex, receiveAddressHex, recoveryAmount);
-        // construct our tx
-        const txBuilder = (getBuilder(this.getChain()) as WrappedBuilder).from(buildTx);
-        // Default expiry is 1 minute which is too short for recovery purposes
-        // extend the expiry to 1 day
-        txBuilder.extendValidTo(RECOVER_TRANSACTION_EXPIRY);
-
-        if (!isUnsignedConsolidations) {
-          const userPrv = this.xprvToCompressedPrv(userKey.toBase58());
-          // receive address only needs to be signed by user key
-          txBuilder.sign({ key: userPrv });
+        let recoveryAmount = 0;
```

Comment:
Now as we are also recovering Tron Tokens here, we will have to use SAFE_TRON_TOKEN_TRANSACTION_FEE in the condition if tokens are to be recovered